layouts/docs/baseof.html

```diff
@@ -5,28 +5,28 @@
   </head>
   <body class="td-{{ .Kind }}">
     <header>
-      {{ partial "navbar-docs.html" . }}
+      {{ partial "navbar.html" . }}
     </header>
     <div class="container-fluid td-outer">
       <div class="td-main">
         <div class="row flex-xl-nowrap">
-          <div class="col-12 col-md-3 col-xl-2 td-sidebar d-print-none">
+          <aside class="col-12 col-md-3 col-xl-2 td-sidebar d-print-none">
             {{ partial "sidebar.html" . }}
-          </div>
-          <div class="d-none d-xl-block col-xl-2 td-toc d-print-none">
+          </aside>
+          <aside class="d-none d-xl-block col-xl-2 td-sidebar-toc d-print-none">
+            {{ partial "page-meta-links.html" . }}
             {{ partial "toc.html" . }}
-          </div>
+            {{ partial "taxonomy_terms_clouds.html" . }}
+          </aside>
           <main class="col-12 col-md-9 col-xl-8 pl-md-5" role="main">
+            {{ partial "version-banner.html" . }}
             {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
             {{ block "main" . }}{{ end }}
           </main>
         </div>
-      </div>  
+      </div>
+      {{ partial "footer.html" . }}
     </div>
-    {{ partial "privacy-notice.html" }}
     {{ partial "scripts.html" . }}
-    {{ $tabsJs := resources.Get "js/tabs.js" }}
-    {{ $tabsJs := $tabsJs | minify }}
-    <script src="{{ $tabsJs.RelPermalink }}"></script>
```

Comment:
I've had a quick scan through the PR locally and I can't tell if these removals are moved to another partial or the layout and content are now different. Given the size of PR would it be ok to quickly summarize the impact of these changes here?